Analysis/include/EventCacheProvider.h

```diff
@@ -117,11 +135,26 @@ class EventCacheProvider {
         bool operator<(const KinFitKey& other) const;
     };
 
+    struct JetScoreKey{
+        size_t jet_index;
+        UncertaintySource unc_source;
+        UncertaintyScale unc_scale;
+
+        JetScoreKey();
+        JetScoreKey(size_t _jet_index,UncertaintySource _unc_source,UncertaintyScale _unc_scale);
+
+        bool operator<(const JetScoreKey& other) const;
+
+        virtual ~JetScoreKey(){} // virtual destructor
+        JetScoreKey( const JetScoreKey & ) = default;
```

Comment:
`~JetScoreKey` and `JetScoreKey( const JetScoreKey & ) = default`  are not needed